src/components/RightSidebar/BreakoutRooms/BreakoutRoomItem.vue

```diff
@@ -148,21 +152,29 @@ export default {
 			return this.roomToken !== this.$store.getters.getToken()
 		},
 
-		roomParticipants() {
-			return this.$store.getters.participantsList(this.roomToken)
-		},
-
 		canFullModerate() {
+			return !this.isParticipantsEditor && (this.participantType === PARTICIPANT.TYPE.OWNER || this.participantType === PARTICIPANT.TYPE.MODERATOR)
 			return this.participantType === PARTICIPANT.TYPE.OWNER || this.participantType === PARTICIPANT.TYPE.MODERATOR
 		},
 
 		canModerate() {
+			if (this.isParticipantsEditor) {
```

Comment:
And i mean to change each new `if (this.isParticipantEditor)` entry in this file. Sorry, if you didn't get it